docs/_includes/common/icon-button.html

```diff
@@ -54,4 +54,48 @@ <h3 id="icon-button-badged">Badged Icon Button</h3>
 </button>
     {% endhighlight %}
 
+    <h3 id="icon-button-cart-nav"> Cart Nav Icon Button </h3>
```

Comment:
I think the wording here is too specific. I believe the general idea is that any icon-button can now have a grey background.

Also, let's move anything icon-button related to a separate PR and keep this PR for tertiary button only.